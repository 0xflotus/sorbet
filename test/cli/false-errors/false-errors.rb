# typed: strict
class K
end

module B
  def foo
  end
end

T.cast(T.unsafe(nil), T.all(K, B)).foo