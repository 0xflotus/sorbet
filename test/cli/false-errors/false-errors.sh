#!/bin/bash

main/sorbet test/cli/false-errors/false-errors.rb -p error-files 2>&1
