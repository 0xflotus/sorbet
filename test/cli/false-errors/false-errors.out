test/cli/false-errors/false-errors.rb
